src/FlatSelect/components/Dropdown/DropdownContext.tsx

```diff
@@ -4,11 +4,11 @@ interface IContext {
 	contextOnKeyDownSearch: (e?: React.KeyboardEvent) => void;
 	contextOnKeyDownItems: (e: React.KeyboardEvent) => void;
 	onSearchRef: (searchElement: React.RefObject<HTMLInputElement>) => void;
-	onItemsRef: (itemElement: React.RefObject<HTMLLIElement>) => void;
+	onItemsRef: (itemElement: React.RefObject<HTMLDivElement>) => void;
 	onFocusElement: (
 		onMouseEnter: () => void,
 		onMouseLeave: () => void,
-		itemElement: React.RefObject<HTMLLIElement>
+		itemElement: React.RefObject<HTMLDivElement>
```

Comment:
Это всё-таки `RefObject`, а не элемент
```suggestion
		itemRef: React.RefObject<HTMLDivElement>
```